eman2/constants.py

```diff
@@ -28,11 +28,11 @@
 """
 
 
-EMAN2DIR = 'EMAN2DIR'
+EMAN2_HOME = 'EMAN2_HOME'
```

Comment:
This can break the backwards compatibility for users setting a custom Eman's home. I agree with the change, but we should document this in some way. Maybe it can be post in the [troubleshooting of Scipion](https://scipion-em.github.io/docs/docs/user/troubleshooting.html).

Reply:
true, I'll put a note on README page